mlserver/kafka/server.py

```diff
@@ -59,12 +61,15 @@ async def start(self):
     async def _consumer_loop(self):
         logger.info("Reading messages from consumer")
         async for request in self._consumer:
-            try:
-                await self._process_request(request)
-            except MLServerError as err:
-                logger.exception(f"ERROR {err.status_code} - {str(err)}")
-            except Exception as err:
-                logger.exception(f"ERROR 500 - {str(err)}")
+            asyncio.Task(self.consume_request(request))
```

Comment:
Should we use the `mlserver.utils.schedule_with_callback` helper instead? 

That way is aligned with other async kick offs of tasks on other parts of the code base. Note that you will need to provide a callback, but this is good practice (and will also get rid of other warnings complaining that a task wasn't awaited). 

Callbacks will generally accept the resolved `Task` object as an argument. Therefore, you could write something along these lines:

```python
def _process_request_cb(self, process_request_task: Task):
	try:
	    process_request_task.result()
	except MLServerError as err:
	    logger.exception(f"ERROR {err.status_code} - {str(err)}")
	except Exception as err:
	    logger.exception(f"ERROR 500 - {str(err)}")
```

Reply:
Thanks,
Good suggestion, I'll make the changes.